NanoProd/python/DiMuon_cff.py

```diff
@@ -1,7 +1,30 @@
+import importlib
+import os
 import sys
 import FWCore.ParameterSet.Config as cms
 from PhysicsTools.NanoAOD.common_cff import CandVars, Var, P3Vars
-from HNL.NanoProd.common_cff import ufloat, uint, ubool
+
+def load(module_file, default_path):
```

Comment:
I don't really understand why this is necessary (but there's probably a good explanation)

Reply:
It is needed for production with crab. crab doesn't preserve the directory structure for transferred files. This could be improved in future, but for now one should use `load` to import noncentral files.

Reply:
Fine - i always thought the files under python would be compiled/copied to $CMSSW_BASE/python, which should in turn be transferred with the compiled local CMSSW, but maybe that doesn't work. In any case it was just a curiosity and we should use what works :-)